src/us/kbase/common/test/service/JsonTokenStreamTest.java

```diff
@@ -559,7 +559,6 @@ private void checkStreamingWithMultiByteChars(Object data, int buffersize, Strin
 	public void detectEncoding() throws Exception {
 		for (String data: basicJsonData) {
 			for (Charset enc: encodings) {
-				@SuppressWarnings("resource")
```

Comment:
Could you please explain more when we need this and why you are deleting it?

Reply:
It just quiets a compiler warning. I'm deleting it because eclipse is telling me it's not needed

Reply:
👍